src/test/java/org/openrewrite/java/migrate/javaee/UseJoinColumnForMappingTest.java

```diff
@@ -0,0 +1,206 @@
+                  private double amount;
+                  
+                  @ManyToOne
+                  @JoinColumn(name="account")
+                  private Account account;
+              }
+              """
+          ),
+          java(
+            """
+              public class Account {
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void noMapping() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import javax.persistence.Entity;
+              import javax.persistence.Column;
+              import javax.persistence.Id;
+              import javax.persistence.ManyToOne;
+                            
+              @Entity
+              public class TransactionEntity {
+                  @Id
+                  private int id;
+
+                  private long transactionNumber;
+                  private double amount;
+
+                  @Column(name="account")
+                  private Account account;
+              }
+              """
+          ),
+          java(
+            """
+              public class Account {
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void noColumn() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import javax.persistence.Entity;
+              import javax.persistence.Column;
+              import javax.persistence.Id;
+              import javax.persistence.ManyToOne;
+                            
+              @Entity
+              public class TransactionEntity {
+                  @Id
+                  private int id;
+
+                  private long transactionNumber;
+                  private double amount;
+
+                  @ManyToOne
+                  private Account account;
+              }
+              """
+          ),
+          java(
+            """
+              public class Account {
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void dropAttributesOnColumnChange() { // and avoid on unaffected @Column
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import javax.persistence.Entity;
+              import javax.persistence.Column;
+              import javax.persistence.Id;
+              import javax.persistence.ManyToOne;
+                            
+              @Entity
+              public class TransactionEntity {
+                  @Id
+                  private int id;
+
+                  private long transactionNumber;
+                  @Column(name="amount", length=512, precision=12, scale=2)
+                  private double amount;
+
+                  @ManyToOne
+                  @Column(name="account", length=512, precision=12, scale=2)
+                  private Account account;
+              }
+              """,
+            """
+              import javax.persistence.*;
```

Comment:
After the recipe ran, there would be 5 imports from `javax.persistence`, and the outputted code used wildcard by default.